packages/nova-react/src/eventing/nova-eventing-provider.test.tsx

```diff
@@ -43,6 +44,7 @@ describe("useNovaEventing", () => {
     prevWrappedEventing = undefined as unknown as NovaReactEventing;
```

Comment:
Let's add a test which is using interceptor and making sure one event is processed and the other moved up